iothub/device/src/IotHubModuleClient.cs

```diff
@@ -163,6 +187,11 @@ public async Task SendMessageToRouteAsync(string outputName, TelemetryMessage me
         /// <exception cref="OperationCanceledException">The operation has been canceled.</exception>
         /// <exception cref="IotHubClientException">An error occured when communicating with IoT hub service.</exception>
         /// <exception cref="ObjectDisposedException">The client has been disposed.</exception>
+        /// <example>
+        /// <code language="csharp">
+        /// await client.SendMessagesToRouteAsync(outputName, new List<TelemetryMessage/> { message1, message2 }, cancellationToken);
```

Comment:
Include client init over AMQP with a comment saying it is only supported over AMQP?